build/linux/dist/install.sh

```diff
@@ -205,6 +205,11 @@ simple_uninstall_f() {
 # Update desktop file and mime databases (if possible)
 updatedbs_f() {
 
+  if [ ! -f "${HOME}/root/.config/mimeapps.list" ]; then
```

Comment:
- `$[HOME}/root/.config/mimeapps.list` folder doesn't look correct to me
- `chmod +x` is wrong, this file is not supposed be executable

Finally, why you create an empty mimeapps.list? how is this supposed to fix #7846?
From what I read in #7846 there is `xdg-mime` that cannot create `/root/.local/share/applications/mimeapps.list.new` becuase the parent directory is nonexistent.

Are you able to reproduce the error reported in the issue #7846 ? If yes, have you tested this fix before submitting?

Reply:
@cmaglie 
yes, I was able to reproduce the error reported, and yes and I did the test before  I submitted my PR, whatever PR I have given for this organization during the GSOC period, have been tested and reproduced the same, so that my PR may be merged efficiently without any problems. I am still working on one of the features in Arduino-cli. Though it might look wrong to you, I have tried my best to tackle that issue I encounter with it and came up with this reasonable solution, which apparently solves the issue.

Reply:
![ar](https://user-images.githubusercontent.com/41021590/80457863-998f1680-894d-11ea-8eab-269445125b3d.png)
and after adding this patch to the code, this is what is written in that particular file, `mimeapps.list`